docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-version = '01.01.01'
+version = '01.01.02'
```

Comment:
+0, okay. I wouldn't bother bumping this; I don't think we rely on it anywhere.

We used to use this `version` flag when we were developing on a monthly/six-monthly/etc. cadence, and back then, we used to also write release notes:

See e.g.

http://lists.openhatch.org/pipermail/devel/2011-April/001838.html

http://blog.openhatch.org/2011/openhatch-0-11-03-is-released/